packages/core/package.nls.json

```diff
@@ -266,7 +267,8 @@
     "AWS.amazonq.codewhisperer.title": "Amazon Q",
     "AWS.amazonq.toggleCodeSuggestion": "Toggle Auto-Suggestions",
     "AWS.amazonq.toggleCodeScan": "Toggle Auto-Scans",
-    "AWS.command.dynamoDb.searchTables": "Search Tables",
+    "AWS.command.dynamoDb.searchTables": "Search DynamoDB Tables",
     "AWS.command.dynamoDb.viewTable": "View Table",
-    "AWS.command.dynamoDb.deleteTable": "Delete Table"
+    "AWS.command.dynamoDb.deleteTable": "Delete Table",
+    "AWS.command.dynamoDb.openTableInConsole": "Open Table in Console"
```

Comment:
I think someone mentioned this in the bug bash, but I wonder if it makes sense to have this be "Open Table in AWS Console"

Reply:
Changed to "Open table in browser"